vistautils/range.py

```diff
@@ -1182,11 +1220,11 @@ def __attrs_post_init__(self) -> None:
             )
 
     @staticmethod
-    def empty() -> "ImmutableRangeMap[K,V]":
+    def empty() -> "ImmutableRangeMap[K, V]":
         return ImmutableRangeMap(ImmutableDict.empty())
```

Comment:
Might as well fix this while we are touching it to:
```suggestion
        return ImmutableRangeMap(immutabledict())
```